tests/Unit/FileSelectionTest.php

```diff
@@ -34,9 +33,10 @@ public function it_can_select_all_the_files_in_a_directory_and_subdirectories()
                 'file1.txt',
                 'file2.txt',
                 'file3.txt',
-            ]),
-            iterator_to_array($fileSelection->selectedFiles())
-        );
+            ]);
+        $selectedFiles = iterator_to_array($fileSelection->selectedFiles());
```

Comment:
Well, that was there since the first time that I clone the repo, and it seems is necessary because the method returns an Generator object.